pkg/daemon/action.go

```diff
@@ -26,12 +27,31 @@ func (pi *policyInstall) String() string {
 	return "install - " + pi.path
 }
 
-func (pi *policyInstall) do(modulePath string, sh semodule.Handler) (string, error) {
+func (pi *policyInstall) do(modulePath string, sh semodule.Handler, ds datastore.DataStore) (string, error) {
+	policyName, err := utils.PolicyNameFromPath(pi.path)
+	if err != nil {
+		return "", fmt.Errorf("installing policy: %w", err)
+	}
 	policyPath, err := utils.GetSafePath(modulePath, pi.path)
 	if err != nil {
 		return "", fmt.Errorf("failed getting a safe path for policy: %w", err)
 	}
-	if err := sh.Install(policyPath); err != nil {
+
+	err = sh.Install(policyPath)
```

Comment:
Because this is the error you want ultimately to be returned, what do you think about swapping the generic err name for installerr and using just err for the db put error? The concern being that if we add another call after the install, we're likely to just use err and rewrite the install error that we later want returned.